src/net.h

```diff
@@ -625,7 +604,7 @@ class NetEventsInterface
 {
 public:
     /** Initialize a peer (setup state, queue any initial messages) */
-    virtual void InitializeNode(CNode* pnode) = 0;
+    virtual void InitializeNode(CNode& pnode, ServiceFlags our_services) = 0;
```

Comment:
This is beginning to feel very mission-creepy, but if you're changing the argument from being a pointer to a ref, then the name of the argument should also change from `pnode` to `node` (p is the prefix for pointer).

Same comment internally in net_processing for the `InitializeNode()` methods.

Reply:
Done.